.github/workflows/release-candidate.deploy.yaml

```diff
@@ -30,7 +31,8 @@ jobs:
     name: Deploy
     runs-on: ubuntu-latest
     environment:
-      name: staging
+      name: release-candidate
+      url: https://staging.app.rootstockcollective.xyz
```

Comment:
I'm unsure if this name can be changed happily. @david-iov might need to implement changes on the infra side before this gets implemented.

(AWS receives this environment name as part of the authorization request and needs to match the one allowed.)

Reply:
I will proceed to update the oidc environment name

Reply:
Thank you @david-iov and @lucasvuotto !

@david-iov while you are in there, is there anyway we can change the subdomain URL to match? So perhaps this workflow file could use the domain: https://release-candidate.app.rootstockcollective.xyz

That way everything matches, and there is less guessing by the devs where things are released to?

Reply:
Can we get a devops request for this changes please? I will work on them asap.